bpf/lib/nodeport.h

```diff
@@ -631,26 +631,26 @@ static inline int nodeport_lb4(struct __sk_buff *skb, __u32 src_identity)
 			return ret;
 	}
 
-	service_port = bpf_ntohs(key.dport);
-	if (service_port < NODEPORT_PORT_MIN ||
-	    service_port > NODEPORT_PORT_MAX) {
-		skb->cb[CB_NAT] = NAT_DIR_INGRESS;
-		skb->cb[CB_SRC_IDENTITY] = src_identity;
-		ep_tail_call(skb, CILIUM_CALL_IPV4_NODEPORT_NAT);
-		return DROP_MISSED_TAIL_CALL;
-	}
-
-	ct_state_new.orig_dport = key.dport;
```

Comment:
I tried to find where we use orig_dport in our code, but looks like the last remains on policy side where removed soe time ago, so we can actually get rid of ct_state's orig_dport in general.